src/metabase/util.cljc

```diff
@@ -283,6 +283,11 @@
     (str (upper-case-en (subs s 0 1))
          (subs s 1))))
 
+(defn kebab-keys
```

Comment:
Isn't `normalize-map` meant to be used instead of this? (I.e., do you really want to preserve strings keys as strings?)

Reply:
I've implemented `kebab-keys`, inspired by `snake-keys`, as used in `annotate/col-info-for-field-clause*`. I was using the function on already keywordized map so I think the usage was fine. On the other hand, thanks for mentioning `nomalize-map`. I've modified the code to use that function. It makes the `kebab-keys` redundant, hence I've removed that.